.github/workflows/font.yml

```
@@ -0,0 +1,43 @@
+name: Build Featherity
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

I think we can move this step after installing the dependencies, right before building the featherity font
```suggestion
      - uses: actions/checkout@v2
```